### DISCUSSION TITLE
Rosuvastatin PBPK model in PKSim

### DISCUSSION THREAD

**Discussion by NileshGaud** [Open-Systems-Pharmacology/Forum#866] [Questions & Problems] [answered]
Hi,

I tried searching for Rosuvastatin (RSV) PBPK model using PKSim platform but could not find anything. 
Did anyone worked on this model using PKSim?
As most of the literature that I reffered to about PBPK modeling of RSV - have the uptake or efflux Clearance parameters reported as Clint while in PKSim we do not have that option for transporters. 
Will it be posible to modify the codes using MoBi accordingly to give input in terms of Clint?

Nilesh

**Comment by NinaHanke** [comment]
Dear Nilesh, 

I am working on a PK-Sim DDI Rosuvastatin model. 
The model is finished, but the publication is not submitted. 
Very sorry, that I cannot share it, yet. 

Yours, 
Nina

**Comment by NileshGaud** [comment]
Dear Nina,

It's great to know that you have developed the model and its finished. I think it will be very useful model.
Can you let me know if possible how much time it may take to get published? 

I had refered to your earlier publications/models, even applied them for understanding DDI potential of project compounds

Thank You

Nilesh

**Comment by NinaHanke** [comment, marked as answer]
Dear Nilesh, 

I really do not want to promise anything. 
The paper still needs a discussion, release by the coauthors, peer review and revision. 
I will do the GitHub upload immediately after acceptance. 

Best, 
Nina

**Comment by StephanSchaller** [comment]
Hi @NinaHanke , do you know the status of this model?

**Reply by NinaHanke** [reply]
Hi Stephan, it is finished but not published, yet.

**Reply by StephanSchaller** [reply]
Thanks for the update, Nina ;-)
